tools/PI/DevHome.PI/ViewModels/SettingViewModel.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using CommunityToolkit.Mvvm.ComponentModel;
+using CommunityToolkit.Mvvm.Input;
+using DevHome.PI.Models;
+
+namespace DevHome.PI.ViewModels;
+
+public partial class SettingViewModel : ObservableObject
```

Comment:
>SettingViewModel [](http://example.com/codeflow?start=21&length=16)

nit: typo, we probably want it as SettingsViewModel.

Reply:
SettingsPageViewModel is the main page for all settings. SettingViewModel is for one setting each time.